### ISSUE TITLE
Tagging JuliaPOMDP Modules

### ISSUE THREAD

**Issue by etotheipluspi** [JuliaPOMDP/POMDPs.jl#103]
One issue with having a single registered package in JuliaPOMDP is that the version the users obtain when they run

``` julia
Pkg.add("POMDPs")
```

is the last tagged version of the package. That's not a problem on its own, because all the tagged versions are working during the time of the tagging. However, if we change a part of the interface in POMDPs.jl, and update the other packages in JuliaPOMDP accordingly (without tagging POMDPs.jl), the user may run into issues between incompatibility of the last tagged version of JuliaPOMDP and the newer versions on the other packages. 

One way to solve this issue is to wait on any changes in other JuliaPOMDP packages until a new tagged version of POMDPs.jl is merged into METADATA.

However, I think we can be more elegant (although at the expense of potentially more work) by tagging all of the packages in JuliaPOMDP as well. Then when a user runs:

``` julia
using POMDPs
POMDPs.add("SARSOP")
```

The SARSOP package that will be fetched is the one that works with the specific version of POMDPs that the user has. That way there are no compatibility issues between the interface and the other packages in the organization. 

What does everyone think?

**Comment by mykelk** [comment]
Sounds sensible.

**Comment by zsunberg** [comment]
Yeah, it seems like we need some mechanism for handling this. I think we
need to sit down and flesh out the details and specify who will be
responsible for what before we commit to something.

**Comment by ChrisRackauckas** [comment]
>is the last tagged version of the package. 

No, it's the highest version set by maximum version requirements. This just means you may need to use maximum version requirements in REQUIRE.

**Comment by etotheipluspi** [comment]
This is a duplicate of #114
